web/src/components/RequestCard/Pagination.tsx

```diff
@@ -0,0 +1,24 @@
+// (c) Tecnologico de Monterrey 2022, rights reserved.
+
+import { useState } from "react";
+
+const Pagination = (): JSX.Element => {
+  const [pageCount, setPageCount] = useState(0);
+
+  const onPageChange = ({ selected }: { selected: number }): void => {
+    // setCount(count + 1)
+  };
+
+  return (
+    <>
+      {/* <Pagination
+        currentPage={pageCount}
+        onPageChange={onPageChange}
+        showIcons={true}
+        totalPages={100}
+        /> */}
+    </>
+  );
+};
+
```

Comment:
Unfinished component? If it is not implemented remove it and add a new issue in the [Project](https://github.com/users/iangg29/projects/2) table.